scripts/build_example

```diff
@@ -91,13 +91,11 @@ if [[ "${docker_image}" == true ]]; then
     exit 1
   fi
 
+  # Docker images only support the Asylo server.
```

Comment:
FYI this may change when #644 is implemented, in that we would use the TLS version of Oak (whether that's asylo or something else)